src/core/ConfigManager.cpp

```diff
@@ -347,18 +346,8 @@ const QString & ConfigManager::value(const QString & cls,
 			}
 		}
 	}
-	static QString empty;
-	return empty;
-}
-
-
-
-const QString & ConfigManager::value(const QString & cls,
-				      const QString & attribute,
-				      const QString & defaultVal) const
-{
-	const QString & val = value(cls, attribute);
-	return val.isEmpty() ? defaultVal : val;
+	
```

Comment:
Found a trailing whitespace here.
```suggestion

```

Reply:
Unimportant, but it might be best if it would just not have the line. I guess the code feels more compact I suppose. If anything, it is one lesser line and doesn't really hurt readability.